editor/components/post-title/index.js

```diff
@@ -110,7 +110,7 @@ class PostTitle extends Component {
 				tabIndex={ -1 /* Necessary for Firefox to include relatedTarget in blur event */ }
 			>
 				{ isSelected && <PostPermalink /> }
-				<h1>
+				<div>
```

Comment:
@jasmussen was this div needed?

Reply:
It was needed in order to let this "pseudo" block inherit styles that look exactly like our other blocks, without writing completely different CSS. 

I did try to apply the on-select border to the textarea itself, or pseudo selectors, but it added a ton of complexity to the CSS, and I don't think I was able to get the responsive behavior to be the same. I decided it wasn't worth a complete rewrite to save this div, and I'm still unconvinced it can be done.